src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx

```diff
@@ -334,6 +344,37 @@ const ActionsPageEvent = ({
                   </span>
                 ),
                 storageSlot: values?.slot?.toHexString(),
+                amountTag: (
+                  <Tag
+                    appearance={{
+                      theme: 'primary',
+                      colorSchema: 'inverted',
+                      fontSize: 'tiny',
+                      margin: 'none',
+                    }}
+                  >
+                    <Numeral
+                      value={values?.stakeAmount || 0}
+                      unit={getTokenDecimalsWithFallback(
+                        colonyNativeToken?.decimals,
+                      )}
+                    />{' '}
+                    {values?.tokenSymbol}
```

Comment:
Please use the numeral suffix

```suggestion
                    <Numeral
                      value={values?.stakeAmount || 0}
                      unit={getTokenDecimalsWithFallback(
                        colonyNativeToken?.decimals,
                      )}
                      suffix={` ${values?.tokenSymbol}`}
                    />
```